src/microbe_stage/Compound.cs

```diff
@@ -30,6 +30,11 @@ public class Compound : IRegistryType
 
     public bool IsEnvironmental;
 
+    /// <summary>
+    ///   Unit for this compound, if applicable (e.g. °C for temperature)
+    /// </summary>
+    public string Unit = null!;
```

Comment:
```suggestion
    public string? Unit;
```
As this can be often null, marking that there is no unit.